web/src/app/chat/components/projects/ProjectChatSessionList.tsx

```diff
@@ -57,10 +56,8 @@ export default function ProjectChatSessionList() {
               onMouseLeave={() => setHoveredChatId(null)}
             >
               <div
-                className={`w-full rounded-xl bg-background-background px-1 py-2 transition-colors ${
-                  hoveredChatId === chat.id
-                    ? "bg-accent-background-hovered"
-                    : ""
+                className={`w-full rounded-08 py-2 transition-colors p-spacing-interline-mini ${
+                  hoveredChatId === chat.id ? "bg-background-tint-02" : ""
```

Comment:
Just a small heads-up: We prefer to use `cn` instead of string formatting in these types of situations!

```ts
className={cn("a", ["b", "c", "d"], myCondition && "e")}
```

The `cn` utility function allows you to nicely merge together a various different types of objects (strings, conditionals, arrays, etc.). It also drops expressions that evaluate to `false` (e.g., `myCondition && ""` will evaluate to `false` if `myCondition` is `false`, but if you don't use `cn`, then `false` will be passed as `"false"` to the Tailwind class!).